test/test_sync.py

```diff
@@ -554,3 +558,78 @@ def test_sync_without_references(mc, project_name: str, config_update: dict):
     sql = "SELECT count(*) FROM notes WHERE ext_url IS NULL"
     gpkg_cur.execute(sql)
     assert gpkg_cur.fetchone()[0] == 3
+
+
+def test_google_drive_backend(mc):
+    """Test media sync connected to google drive backend"""
+    project_name = "mediasync_test_googledrive"
+    full_project_name = WORKSPACE + "/" + project_name
+    work_project_dir = os.path.join(TMP_DIR, project_name + "_work")
+
+    cleanup(mc, full_project_name, [work_project_dir])
+    prepare_mergin_project(mc, full_project_name)
+
+    # patch config to fit testing purposes
+    config.update(
```

Comment:
can you add some tests for invalid config?

Reply:
added